cpp/modmesh/toggle/RadixTree.hpp

```diff
@@ -72,6 +73,11 @@ class RadixTreeNode
     const T & data() const { return m_data; }
     const child_list_type & children() const { return m_children; }
 
+    bool no_children() const
```

Comment:
There are two options for this getter: `no_children()` or `has_children()`.

`no_children()` acts like the implementation detail `m_children.empty()`.  If we choose this, what is the reason that we do not name the function as `empty_children()`?  `empty` is more consistent to the style of STL.

Please consider the options and we should discuss.

Reply:
I agree that no_chlidren should be renamed to empty_children, no_children is too verbal.
But I am curios, in practical, how to choose between negative check and positive check?
like empty_children or has_children in this case

Reply:
Time will tell.